web/packages/api/src/status.ts

```diff
@@ -72,8 +72,8 @@ export const BlockLatencyThreshold = {
 }
 
 export const InsufficientBalanceThreshold = {
-    // Minimum as 10 DOT
-    Substrate: 100_000_000_000,
+    // Minimum as 100 DOT
+    Substrate: 1_000_000_000_000,
```

Comment:
Let's make it 300 DOT.

Reply:
https://github.com/Snowfork/snowbridge/pull/1249/commits/3c60b685a6024edb5e377a51bdcaf87fb0118d18